src/pages/Search/components/Form.styles.js

```diff
@@ -0,0 +1,42 @@
+import styled, { css } from 'styled-components';
+import button from '../../../components/button';
```

Comment:
The styling can be simplified a bit. Not sure why this works and yours is more complicated with `margin-left: -1px` and so on...

- FormContainer gets `display: flex; flex-direction: column; align-items: center;` to center align the headline and form
- InputGroup gets `margin-top: 29px` and no margin-left
- Prefixed and Input get `margin-right: 10px`
- Button gets `height: 36px; padding: 0px 16px;`